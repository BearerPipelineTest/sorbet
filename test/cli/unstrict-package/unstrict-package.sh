cd test/cli/unstrict-package || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages . 2>&1
