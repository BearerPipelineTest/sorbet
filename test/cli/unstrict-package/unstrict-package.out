foo/foo.rb:12: Unable to resolve constant `BardClass` https://srb.help/5002
    12 |    Project::Bar::BardClass
            ^^^^^^^^^^^^^^^^^^^^^^^
  Autocorrect: Use `-a` to autocorrect
    foo/foo.rb:12: Replace with `Project::Bar::BarClass`
    12 |    Project::Bar::BardClass
            ^^^^^^^^^^^^^^^^^^^^^^^
    bar/bar.rb:5: Did you mean: `Project::Bar::BarClass`?
     5 |  class BarClass
          ^^^^^^^^^^^^^^
Errors: 1
